tailwind.config.js

```diff
@@ -17,7 +17,8 @@ module.exports = {
 
         blue: '#384aff',
         green: '#21b66f',
-        red: '#ff3856'
+        red: '#ff3856',
+        purple: colors.violet
```

Comment:
Wasn't this already removed and instead changed in [State.vue](https://github.com/bigint/snapshot/blob/f27228c45edfa313d9a42fc793003845ab5af3cc/src/components/Ui/State.vue#L10)? I think that makes more sense, since it's the only place we use that color.